### DISCUSSION TITLE
Where can I find community living type roommate sites?

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#801] [Research]
I am looking to have a house that is more cooperative living. Ideas like group meetings, land plots for growing food if one wanted and a say about what was happening in the house and to the house have been talked about. Maybe even all co-sharing cars and the costs.

My problem is finding the people. I’m also not finding any community living type roommate sites. I’m wondering how to find someone who is looking for this type of living?

**Comment by rufuspollock** [comment]
@laurenwigmore great question - and i like the specifics people are interested in.

**Comment by rufuspollock** [comment]
I'd start with https://www.ic.org/directory/ - the one issue is that it is a bit overwhelming and coverage outside of the US (e.g. in Europe) is a bit limited.
